SwiftMonkeyPaws/MonkeyPaws.swift

```diff
@@ -112,14 +124,16 @@ public class MonkeyPaws: NSObject, CALayerDelegate {
                 gesture.extend(to: point)
             }
         } else {
-            if gestures.count > maxGesturesShown {
+            if gestures.count > self.configuration.paws.maxShown {
```

Comment:
`self` not needed.

Reply:
My rule of thumb with the usage of `self` is if a variable declared on the scope of the class, it must have the `self` prefix. But i understand you guys have a different code style, happy to adjust.

Reply:
This comes more from using SwiftFormat that automatically removes or extra `self` usages. I must definitely include it somehow in the build process so that the consistency check is done (and applied) automatically.